connectors/sources/mysql.py

```diff
@@ -123,13 +86,14 @@ def __init__(self, configuration):
         """Set up the connection to the MySQL server.
 
         Args:
-            configuration (DataSourceConfiguration): Object of DataSourceConfiguration class.
+            configuration (DataSourceConfiguration): Object  of DataSourceConfiguration class.
```

Comment:
Interesting, where this space comes from...I'll check, if that was our autoformat.